lib/rspec/matchers.rb

```diff
@@ -135,11 +135,11 @@ module RSpec
   #       end
   #
   #       def failure_message
-  #         "expected #{@target.inspect} to be in Zone #{@expected}"
+  #         "expected #{RSpec::Support::ObjectInspector.inspect(@target)} to be in Zone #{@expected}"
```

Comment:
We've officially said that `RSpec::Support` is not part of RSpec's public API.  That may change at some point, but we're not yet ready to commit to maintaining its APIs according to SemVer policies.  Given that, I'd rather our docs not show it's use.